### DISCUSSION TITLE
Add Firefox to container

### DISCUSSION THREAD

**Discussion by htudor** [jlesage/docker-jdownloader-2#194] [Ideas]
Hello
Can you add Firefox for captchas and other web interactions?

**Comment by mihawk90** [comment]
You can install packages manually using [`INSTALL_PACKAGES`](https://github.com/jlesage/docker-jdownloader-2#:~:text=0-,INSTALL_PACKAGES,-Space%2Dseparated%20list). But that said, I tried this, and I cannot get Firefox to open after setting the command line for it in the advanced settings (see [JD Support article](https://support.jdownloader.org/en/knowledgebase/article/define-custom-browser-solver-captcha-browser)). Firefox only throws "Your Firefox profile cannot be loaded. It may be missing or inaccessible." (even after adding a `--profile` to a writable location), and Chromium doesn't bring up anything at all.

If you have more luck, I'd like to see how.
